package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "src/index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest --coverage --collectCoverageFrom=src/**/*.{js,jsx}",
```

Comment:
Configure this to collect coverage from all files under `src/test/*.{js, jsx}`

Reply:
I think the logic behind the coverage is to collect coverage of tested files, not the files used to test those files. If I do it in the suggested way It means that I want to collect the coverage of the tests file not the file I am testing. The result on the picture below
**Table 1:** `jest --coverage --collectCoverageFrom=src/**/*.{js,jsx}` coverage of project files [test file excluded]
**Table 2:** `jest --coverage --collectCoverageFrom=src/__test__/*.{js,jsx}` coverage of test files
![image](https://user-images.githubusercontent.com/29593858/58953153-e6965700-8795-11e9-8879-af81e6d2b3d4.png)

Reply:
What I meant is that, in my previous comment, I suggested us to make use of `test` folder instead of `__test__`. 
That is why I suggested `jest --coverage --collectCoverageFrom=src/test/*.{js,jsx}` and not `jest --coverage --collectCoverageFrom=src/__test__/*.{js,jsx}`

Reply:
We can put tests in one folder.

Reply:
@me-x-mi 
Ohhhh I understand.
but I tried to change the folder name and it brings the same result as shown on the image from my previous comment.

Reply:
I coverage table I am getting no file yet I have the files that I am testing

Reply:
@me-x-mi 
But this shows what we are expecting to get

![image](https://user-images.githubusercontent.com/29593858/58955080-3fb4b980-879b-11e9-80b6-4ce27533638a.png)

Reply:
You have not changed this folder/file. 
That is why such error occurs.

<img width="847" alt="Screen Shot 2019-06-05 at 17 03 18" src="https://user-images.githubusercontent.com/48981919/58962467-e1e09b80-87b3-11e9-9e44-5879667e8411.png">

Reply:
@me-x-mi 
OOhhhh I haven't pushed yet because we didn't decide which way are we going to stay with.